project/src/main/webapp/event-details.html

```diff
@@ -28,6 +28,26 @@
     <div class="container">
       <div class="card">
         <div class="card-body">
+        <form
+          id="image-form"
+          method="POST"
+          enctype="multipart/form-data"
+        >
+          <div class="col-sm-3">
+            <img id="event-picture" />
+            <br /><br />
+            <div class="form-group">
+              <label for="event-picture">Upload an image for your event: </label>
+              <br />
+              <div class="form-group">
+                <div class="input-group">
+                  <input type="file" name="event-picture" />
```

Comment:
want to center within div

Reply:
I'll make these UI changes on top of your branch in a separate PR so I'm not working on old displays